index.html

```diff
@@ -143,6 +143,16 @@ <h1>Lambda Times</h1>
       </div>
     </div>
 
+    <div class="card" data-tab="node">
+      <div class="headline">Hey: LOOK AT ME...I'M THE ADDED CARD</div>
+      <div class="author">
+        <div class="img-container">
+          <img src="/Sprint-Challenge.gif" />
```

Comment:
When you put the slash at the beginning, it becomes an absolute path. This will work if you run it on most servers, but trying to run it without a server causes the image to not show up. Removing the slash will likely allow it to work everywhere.